src/backends/perf_pt/decode.c

```diff
@@ -173,8 +173,8 @@ perf_pt_init_block_decoder(void *buf, uint64_t len, int vdso_fd, char *vdso_file
  * Updates `*first_instr` and `*last_instr` with the address of the first and last
  * instructions of the next block in the instruction stream.
  *
- * An instruction address of 0 indicates that the end of the instruction stream
- * has been reached.
+ * If the returned instruction addresses are 0, this indicates that the end of
```

Comment:
A user might have the question: do I have to check both are zero, or is it sufficient to check only one?

Reply:
Fixed.